### DISCUSSION TITLE
Creating a code with a logo

### DISCUSSION THREAD

**Discussion by m-gallesio** [Shane32/QRCoder#323] [Q&A] [answered]
Greetings.
Does this library currently support creating codes with a logo at the center?
This is what I mean: https://commons.wikimedia.org/wiki/File:QR_code_Wi-Fi.png

**Comment by codebude** [comment, marked as answer]
Hi @m-gallesio ,

yes, it does. There is a overload of the GetGraphic-method that accepts a logo graphics. For more information checkout our wiki, especially this section: https://github.com/codebude/QRCoder/wiki/Advanced-usage---QR-Code-renderers#21-qrcode-renderer-in-detail (See: "Overload 4").

If you need further assistence, let us know.

**Reply by m-gallesio** [reply]
I thank you for your swift answer and blame myself for not reading the documentation thoroughly enough.
